script_library/run-ansible.sh

```diff
@@ -4,21 +4,6 @@
 
 function run_ansible(){
     set -x
-    # ansible-runner default locations
```

Comment:
I think we should keep this, as the bootstrap and the run_ansible functions can be dissociated. When not installing, ANSIBLE_RUNNER_DIR won't be set.

Reply:
I don't understand the use case you mean. Is there a case where we use 'script_library/run-ansible.sh' directly instead of being called by 'run.sh'? I thought it was only used by 'run.sh' which always checks ANSIBLE_RUNNER_DIR for a usable ansible binary.

Reply:
1) yes for debugging purposes, one might want to source run-ansible.
2) install_ansible function is only sourced when install_ansible needs to happen. When things don't need to happen (ansible already installed), ANSIBLE_RUNNER_DIR won't be set.

Reply:
If you want, I ahve a different approach in https://github.com/evrardjp/socok8s/commits/scrd-4754-part2 , and it should solve your issue.